### PR TITLE
[MIG] stock_location_bin_name: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**

### CODE REVIEW

**Review by sebalix — APPROVED** [review]
Lacking of some tests but LG

**Review by TDu — APPROVED** [review]

**Review by mmequignon — APPROVED** [review]

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo14-addon-stock_location_zone @ git+https://github.com/phuctranerp/stock-logistics-warehouse.git@14.0-mig-stock_location_zone#subdirectory=setup/stock_location_zone
```
Comment (simahawk) [inline test-requirements.txt]: please remove these once #1077  is merged. Also, next time, please link the depending PRs in the description ;)

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by hailangvn** [comment]
Thanks @simahawk. This PR has been updated.

**Review by simahawk — APPROVED** [review]

File: stock_location_bin_name/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "name": "Stock Location Bin Name",
+    "version": "14.0.1.0.0",
+    "author": "BCIM, Okia, Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "summary": "Compute bin stock location name automatically",
+    "category": "Stock Management",
+    "depends": ["stock_location_zone", "stock_location_position"],
+    "data": ["views/stock_location.xml"],
+    "installable": True,
+    "development_status": "Alpha",
```
Comment (simahawk) [inline stock_location_bin_name/__manifest__.py]: I think we can move to Beta but then we'll need some tests to move to Stable

**Pull request merged**